packages/modules/order/src/utils/transform-order.ts

```diff
@@ -223,3 +224,35 @@ export function mapRepositoryToOrderModel(config, isRelatedEntity = false) {
 
   return conf
 }
+
+function mapRepositoryToRelatedEntity(config) {
```

Comment:
suggestion: I think a small comment in the code about what this is for would be helpful for the next one seeing it. It was not 100% for me at first glance

Reply:
Will add some comments on this entire file and explain these functions.